.travis.yml

```diff
@@ -0,0 +1,35 @@
+sudo: false
+language: node_js
+cache:
+  directories:
+    - node_modules
+notifications:
+  email: false
+node_js:
+  - 0.12
+  - 4
+  - 5
```

Comment:
You can get rid of Node 5. It [reached end of life](https://nodejs.org/en/blog/community/v5-to-v7/) a while back and we no longer support it.